server/controllers/request.js

```diff
@@ -0,0 +1,99 @@
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const Request = require("../models/Request");
+
+exports.createRequest = async (req, res, next) => {
+    const { ownerId, sitterId } = req.body;
```

Comment:
Optional: We can also validate the request body params before passing them to mongoose queries